app/Console/Commands/Install.php

```diff
@@ -248,9 +248,14 @@ protected function formatTableCells(array $cells, $color)
      */
     protected function getValidDbDrivers()
     {
-        return array_keys(array_filter($this->dbDrivers, function ($item) {
-            return $item === true;
-        }));
+        return array_map(
+                function ($pdo_name){
+                    return substr($pdo_name, strlen('pod_'));
```

Comment:
Strictly speaking, it should be `pdo` here (but it doesn't change the length ;-)